cluster-tag-filter (#1059)
spectrocloud/librarium: content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md

---

> title: "Palette Resource Roles"

Reviewer: ```suggestion
title: "Global and Custom Resource Roles"
```

> metaTitle: "Palette Global and Custom Resource Roles "

Reviewer: ```suggestion
metaTitle: "Global and Custom Resource Roles"
```

> metaDescription: "Palette Global and Custom Resource Roles "

Reviewer: ```suggestion
metaDescription: "Palette contains global resource roles and supports the ability to create custom resource roles. "
```

hideToC: false
fullWidth: false
---

import Tabs from 'shared/components/ui/Tabs';
import WarningBox from 'shared/components/WarningBox';
import InfoBox from 'shared/components/InfoBox';
import PointsOfInterest from 'shared/components/common/PointOfInterest';
import Tooltip from "shared/components/ui/Tooltip";

# Overview

> The page encompasses the Palette Resource Roles. We support two broad classifications of Resource Roles:

Reviewer: ```suggestion
Palette support two types of resource roles, global resource roles and custom resource roles:
```
Use simple terms.

<br />

> * [Palette Global Resource Roles](/clusters/cluster-management/cluster-tag-filter/global-cutome-rroles#globalresourceroles), the set of roles built in and available to you.

Reviewer: ```suggestion
* [Palette Global Resource Roles](/clusters/cluster-management/cluster-tag-filter/global-cutome-rroles#globalresourceroles) are a set of roles built in and available to you.
```

> * [Palette Custom Resource Roles](/clusters/cluster-management/cluster-tag-filter/global-cutome-rroles#customroles), the roles you can create in Palette using our available set of permissions and operations.

Reviewer: ```suggestion
* [Palette Custom Resource Roles](/clusters/cluster-management/cluster-tag-filter/global-cutome-rroles#customroles), are roles you can create in Palette using a set of permissions and operations.
```

>    * [Create Custom Role](/clusters/cluster-management/cluster-tag-filter/global-cutome-rroles#createcustomroles), how to create a custom role in the Palette console.

Reviewer: ```suggestion

To learn how to create a custom role. Review the [Create Custom Role](/clusters/cluster-management/cluster-tag-filter/global-cutome-rroles#createcustomroles) guide.
```

| **cloudconfig**    |            | √          | √       | √        | √          |            |             |            |             |
| **cluster**        |            | √          | √       | √        | √          |            |             |            |             |
| **clusterProfile** |            | √          | √       | √        | √          |            | √           |            |             |
| **dnsMapping**     |            |            | √       | √        |            |            |             |            |             |
| **location**       |            |            | √       | √        |            |            |             |            |             |
| **machine**        |            |            | √       | √        |            |            |             |            |             |
| **macro**          |            |            | √       | √        |            |            |             |            |             |
| **packRegistry**   |            |            | √       | √        |            |            |             |            |             |

## Resources

> [Resource Scope Matrix](http://localhost:9000/user-management/palette-rbac#resourcescopematrix)

Reviewer: ```suggestion
[Resource Scope Matrix](/user-management/palette-rbac#resourcescopematrix)
```
resma, I shouldn't have to fix simple issues like this. Slow down and review your changes.
